app/Http/Controllers/ApplicationController.php

```diff
@@ -82,30 +83,29 @@ public function store(Request $request)
 
     // @TODO: there's a better way of doing the following...
     $application = new Application();
-      $application->accomplishments = $request['accomplishments'];
+      $application->accomplishments = Request::get('accomplishments');
```

Comment:
This is a bit subjective, but I'd prefer to use the passed request object here like `$request->get('accomplishments')` rather than using the facade.